packages/ui-react/src/containers/AccountModal/forms/Checkout.tsx

```diff
@@ -223,6 +93,19 @@ const Checkout = () => {
     );
   }
 
+  const cancelUrl = createURL(window.location.href, { u: 'payment-cancelled' });
+  const waitingUrl = createURL(window.location.href, { u: 'waiting-for-payment' });
+  const errorUrl = createURL(window.location.href, { u: 'payment-error' });
```

Comment:
Not sure, I think they depend on the current domain and pathname or do you mean just the `u` constants?

Reply:
Yeah, just `u` constant.

Reply:
Can we do that in a separate PR? Because these `u` constants are used a lot (~27 files and 82 occurrences 😅)

Reply:
Of course!

Reply:
In most cases we use this util, which is typed from `AccountModals`: 
```javascript 
const modalURLFromLocation = (location: Location, u: keyof AccountModals | null, queryParams?: QueryParamsArg) => string
```

However, in this case we need the full url (including hostname), because they're used as return url's, so I decided to use the `createURL` util.

We could create a wrapper util to make it more strict. What do you think: @AntonLantukh 

```javascript
const createAccountURL = (u: keyof AccountModals) => createURL(window.location.href, { u });
```

Reply:
Perfect! I suggest we merge this first (as Christiaan suggested) while I'll implement it into our current working branch.